src/SFA.DAS.Commitments.Domain/Entities/Commitment.cs

```diff
@@ -17,7 +17,7 @@ public Commitment()
         public string LegalEntityId { get; set; }
         public string LegalEntityName { get; set; }
         public string LegalEntityAddress { get; set; }
-        public OrganisationType LegalEntityOrganisationType { get; set; }
+        public SFA.DAS.Common.Domain.Types.OrganisationType LegalEntityOrganisationType { get; set; }
```

Comment:
This is explicitly named here to avoid accidental and contamination with the still available, but now obsolete, local enum of the same name